### DISCUSSION TITLE
Smoking Motor Issue

### DISCUSSION THREAD

**Discussion by christopherbarbarie** [dannyjacobs/ECHO#56] [General]
A few days ago I took our new drone (#2) out to do its first mission mode flight. It was intended to takeoff, gain 20m in altitude, go forward and come back at the same altitude and then land. As soon as it took off however, it started bouncing up and down off the ground, I let it run for a bit thinking it might takeoff and do its mission. A few seconds later one of the motors started smoking, I took over manually and landed it immediately. Luckily, everything seems to still be working other than the coils in the motor being a little black, I test spun all the motors and took it out for another manual flight and nothing seems wrong. 

I grabbed the ulogs from the bad mission mode flight and the test manual flight after that and compared the actuator output for the motor that had smoke coming out. 

This is the output for the nominal manual flight:
![output_1_good](https://user-images.githubusercontent.com/106403013/184681249-cfa750ab-492f-448b-ae2f-2c00c28ac076.png)

This is the output for the mission mode flight:
![output_1_bad](https://user-images.githubusercontent.com/106403013/184681328-6d2d3d40-e4ba-466d-9625-fc3ea32a0221.png)

It seems like the output of the motor was maxing out the entire time of the flight up until it was switched into position mode and landed.

**Comment by dannyjacobs** [comment]
This is happened to us before when we got some schmutz in the motor. If there’s any ferrus material in the sand or soil around it’ll get attracted to the magnets and motor. 

Interesting that you blew the motor not the ESC.

Now I always check with my hand  to make sure that all of the motors turn freely before flying.

**Reply by christopherbarbarie** [reply]
Interesting I was flying it in a sandpit at the time so very possible something got stuck in the motor. Would this cause the motor to stop spinning? I feel like I would have seen it if the propeller wasn't spinning at the time

**Comment by dannyjacobs** [comment]
We had this happen out at the Owens Valley site. It is also sandy and known
to have a small amount ferrous material.  In our case the props would spin
ok most of the time but occasionally the prop would visibly spin less fast.
Hard to see visually in some cases. These motors were also almost six years
old at that point. I bet things might be a bit different with more powerful
motors and ESCs.  I've also had ESCs fail and go on fire, presumably for
similar high current reasons.





On Tue, Aug 16, 2022 at 6:44 AM christopherbarbarie <
***@***.***> wrote:

> Interesting I was flying it in a sandpit at the time so very possible
> something got stuck in the motor. Would this cause the motor to stop
> spinning? I feel like I would have seen it if the propeller wasn't spinning
> at the time
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dannyjacobs/ECHO/discussions/56#discussioncomment-3406756>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPNVZKWIMJADFXW4ITUDTVZOLNPANCNFSM56S3J5LQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Arizona State University
School of Earth and Space Exploration
Low Frequency Cosmology
Phone:           (505) 500 4521
Homepage:     danielcjacobs.com

**Reply by christopherbarbarie** [reply]
Ok, I'm thinking this seems very similar to our issue, how have you been dealing with this? Simply avoiding takeoff and landing near sand?

**Comment by dannyjacobs** [comment]
Yeah, avoiding sand and checking frequently for possible crunchers in the
rotors.

One thing we've tried to do is have a decent sized landing pad. At OVRO we
borrowed a big old exercise mat. In Greenbank we had a great lump of old
plywood.  At Sevilleta station we had a tarp which was miserably small and
flappy.

Another thing to do is do a little survey for magnetics at your launch
site. Get a big old neodymium or similar, drag it through the sand, and see
what you get. Once you do this a few times at different places you'll get
an idea of how much caution will be warranted. This, by the way, is how
they survey sites for archaeological evidence of early industry. All those
little sparks coming off the ironmongery add up. Maybe you'll find yourself
an ancient forge.

On Wed, Aug 17, 2022 at 11:43 AM christopherbarbarie <
***@***.***> wrote:

> Ok, I'm thinking this seems very similar to our issue, how have you been
> dealing with this? Simply avoiding takeoff and landing near sand?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dannyjacobs/ECHO/discussions/56#discussioncomment-3417130>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPNVYR2X4UTPCUMHLWCK3VZUXDJANCNFSM56S3J5LQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Arizona State University
School of Earth and Space Exploration
Low Frequency Cosmology
Phone:           (505) 500 4521
Homepage:     danielcjacobs.com

**Reply by christopherbarbarie** [reply]
Hadn't considered a landing pad, that seems like a good thing for us to have as well for our deployments. Flying in the sand hadn't been a problem in Montreal hahahah
